src/app/clusters/on-off-server/on-off-server.cpp

```diff
@@ -452,12 +452,12 @@ Status OnOffServer::setOnOffValue(chip::EndpointId endpoint, chip::CommandId com
                 ChipLogProgress(Zcl, "ERR: writing on/off %x", to_underlying(status));
                 return status;
             }
+        }
 
-            if (SupportsLightingApplications(endpoint))
-            {
-                ChipLogProgress(Zcl, "Off completed. reset OnTime to  0");
-                Attributes::OnTime::Set(endpoint, 0); // Reset onTime
-            }
+        if (SupportsLightingApplications(endpoint))
+        {
+            ChipLogProgress(Zcl, "Off completed. reset OnTime to  0");
```

Comment:
If you move this outside the "else { // off" branch, the comment probably needs to update as well.

Reply:
Never mind - was mis-counting brackets. This method is not friendly to review :/